content/en/samples/posts/2024-12-26-idle-view-app.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Idle View App for webOS Devices
+date: 2024-12-26
+slug: idle-view-app
+posttype: app
+toc: true
+thumbnail: th-idle-view-app.png
+---
+
+**Author: Kangin Lee, Seungjin Kim, Tutku Genis**
+
+## Table of Contents
+
+1. [Overview of Project](#overview-of-project)
+2. [Key Features](#key-features)
+3. [Hardware Requirements](#hardware-requirements)
+4. [Software Requirements](#software-requirements)
+5. [Project Setup](#project-setup)
+   - [Backend Setup](#backend-setup)
+   - [Frontend Setup](#frontend-setup)
+6. [Deployment to webOS](#deployment-to-webos)
+7. [Additional Notes for Packaging and Installing Apps](#additional-notes-for-packaging-and-installing-apps)
+8. [Code Implementation](#code-implementation)
+9. [Contact](#contact)
+
+## Overview of Project
+
+**The Personalized IdleView Application** for webOS aims to revolutionize idle screen functionality by delivering meaningful, real-time information tailored to each user. Designed with energy efficiency as main principle, this app minimizes power consumption while ensuring consistent delivery of real-time, customizable content.
+
+<p align="center"> <img src="https://github.com/ttkgns/Idle-View-team7/blob/main/dashboard_heritage_red.png?raw=true" alt="alt text" /> </p>
+
+<p align="center">
+  <img src="https://github.com/ttkgns/Idle-View-team7/blob/main/brightness_control_settings.png?raw=true" alt="Image 1" width="49%" />
+  <img src="https://github.com/ttkgns/Idle-View-team7/blob/main/background_change_widget.png?raw=true" alt="Image 2" width="49%" />
+</p>
+                                                                            
+
+
+### Key Features
```

Comment:
You should refer to resources such as detailed diagrams, infographics, or explanatory images to better understand how each feature functions.